packages/skeleton-svelte/src/lib/internal/uuid.ts

```diff
@@ -0,0 +1,7 @@
+// UUID
```

Comment:
a UUID is a spec for a type of ID, we should not name this `uuid` or even mention it, this is an id, not anything else.